membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -220,10 +222,59 @@ def universe_dummy(self):
                       [0., 200., 150.], [100., 200., 150.], [200., 200., 150.]])
 
         u = mda.Universe(a, n_atoms=9)
-        u.dimensions = [3, 3, 3,  90., 90., 90.]
+        u.dimensions = [300, 300, 300,  90., 90., 90.]
+
+        return u
+
+    @pytest.fixture()
+    def universe_dummy_full(self):
+        a = np.array([[0., 0., 150.], [100., 0., 150.], [200., 0., 150.],
+                      [0., 200., 150.], [100., 200., 120.], [200., 200., 120.],
+                      [0., 100., 120.], [100., 100., 120.], [200., 100., 120.]])
+
+        u = mda.Universe(a, n_atoms=9)
+        u.dimensions = [300, 300, 300,  90., 90., 90.]
 
         return u
 
+    @pytest.fixture()
+    def universe_dummy_wrap(self):
+        #   +-----------+
+        #   |   | 7 | 8 | 9
+        #   +-----------+
+        # 4 | 5 | 6 |   |
+        #   +-----------+
+        #   |   | 1 | 2 | 3
+        #   +-----------+
+        a = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
+                      [0., 100., 150.], [100., 100., 150.], [-100., 100., 150.],
+                      [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])
+
+        u = mda.Universe(a, n_atoms=9)
+        u.dimensions = [300, 300, 300,  90., 90., 90.]
+
+        return u
+
+    # Equivalent to universe_dummy_wrap when wrapping is applied.
+    # Atoms out of bounds in x and y
+    @pytest.fixture()
+    def universe_dummy_wrap_xy(self):
+        a = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
+                      [0., -200., 150.], [100., -200., 150.], [-100., 100., 150.],
+                      [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])
```

Comment:
The 300s will convert to 0 unless you change the bounds to 300.01 or similar, so I don't think it'll be equivalent? Also, suggestion: would `dummy_universe_unwrapped_xy` be better, because this is *not* wrapped?

Reply:
I'm not quite sure what this does that `universe_dummy_wrap` does not, actually, since `universe_dummy_wrap` also wraps to different coordinates?

Reply:
Sorry if I wasn't clear. Both universes are unwrapped. When they wrap, they are equivalent. Then there is a test where we can check that coordinates wrap correctly in x, and another separate that wraps correctly in x and y.
```
import MDAnalysis as mda
import numpy as np

array_x = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
              [0., 100., 150.], [100., 100., 150.], [-100., 100., 150.],
              [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])

u = mda.Universe(array_x, n_atoms=9)
u.dimensions = [300, 300, 300,  90., 90., 90.]
ag2 = u.atoms.wrap()
ag2
array([[  0.,   0., 110.],
       [100.,   0., 150.],
       [200.,   0., 150.],
       [  0., 100., 150.],
       [100., 100., 150.],
       [200., 100., 150.],
       [  0., 200., 110.],
       [100., 200., 150.],
       [200., 200., 150.]], dtype=float32)
```
and
 
```
array_xy = np.array([[300., 0., 110.], [100., 0., 150.], [200., 0., 150.],
              [0., -200., 150.], [100., -200., 150.], [-100., 100., 150.],
              [300., 200., 110.], [100., 200., 150.], [200., 200., 150.]])

u = mda.Universe(array_xy, n_atoms=9)
u.dimensions = [300, 300, 300,  90., 90., 90.]
ag2_xy = u.atoms.wrap()
ag2_xy

ag2_x == ag2_xy
array([[ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True],
       [ True,  True,  True]])
```
If that's ok, I'd prefer to keep both cases. I think it's clearer. In the first case, only atoms are out of bounds in the x coordinate, while in the second atoms are out of bounds in x and y.